varats/varats/tools/tool_util.py

```diff
@@ -64,3 +65,19 @@ def get_research_tool(
 def get_supported_research_tool_names() -> tp.List[str]:
     """Returns a list of all supported research tools."""
     return ["phasar", "vara"]
+
+
+def configuration_lookup_error_handler(func) -> tp.Any:
+    """Wrapper for drivers to catche internal Exceptions and provide a helpful
+    message to the user."""
+
+    def wrapper_configuration_lookup_error_handler(*args, **kwargs):
```

Comment:
```suggestion
    @wraps(func)
    def wrapper_configuration_lookup_error_handler(*args, **kwargs):
```

See here for an explanation: https://stackoverflow.com/questions/308999/what-does-functools-wraps-do